qp_klp/tests/data/configuration_profiles/miseq_metagenomic.json

```diff
@@ -64,7 +84,7 @@
         ],
         "fastqc_executable_path": "fastqc",
         "multiqc_executable_path": "multiqc",
-        "multiqc_config_file_path": "sequence_processing_pipeline/multiqc-bclconvert-config.yaml",
+        "multiqc_config_file_path": "/home/user/qp_klp/tests/data/multiqc-bclconvert-config.yaml",
```

Comment:
This should actually be an issue, no?

Reply:
Not sure I understand -  it's just a dummy path.

Reply:
correct.